dynamic-plugins.default.yaml

```diff
@@ -538,21 +538,14 @@ plugins:
 
   # Group: Global floating action button
   - package: ./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-global-floating-action-button
-    disabled: true
+    disabled: false
```

Comment:
by default won't this be disabled?

Reply:
This plugin doesn't render anything by default. It "waits" that other plugins to use it. So this is fine.